gnd/src/androidTest/java/com/google/android/gnd/system/auth/FakeAuthenticationManager.java

```diff
@@ -27,6 +27,7 @@
 
   private static final String TAG = FakeAuthenticationManager.class.toString();
 
+  @Hot(replays = true)
```

Comment:
Missing import for this -- that's why the build failed.

Reply:
Good catch!!! We were wondering.. I think AS got out of sync somehow, and I didn't realize auto-import didn't work. Thanks for finding this!